unittest/CMakeLists.txt

```diff
@@ -73,6 +73,15 @@ function(add_python_eigenpy_lib_unit_test name source)
   set(test_target ${PROJECT_NAME}-${name})
   add_python_unit_test(${test_target} ${source} "lib" "bin" "python")
   set_tests_properties(${test_target} PROPERTIES DEPENDS ${PYWRAP})
+  string(REPLACE "unittest/" "" source_rel "${source}")
+  add_custom_target(
+    "test-${name}"
+    ${CMAKE_COMMAND}
+    -E
+    env
+    PYTHONPATH=${PROJECT_BINARY_DIR}/lib:${PROJECT_BINARY_DIR}/python:$ENV{PYTHONPATH}
+    ${PYTHON_EXECUTABLE}
+    "${CMAKE_CURRENT_SOURCE_DIR}/${source_rel}")
```

Comment:
```suggestion
```

You can remove this code. `add_python_unit_test` is already setting the appropriate PYTHONPATH to the test.